.buildkite/browser-pipeline.full.yml

```diff
@@ -29,7 +29,6 @@ steps:
         run: browser-maze-runner-bs
         use-aliases: true
         command:
-          - --https
```

Comment:
i noticed this wasn't working for the ios mobile devices (but it does pass for anrdoid). Wasn't sure it was worth splitting this so that we run over https on android so ive just disabled https for all mobile device runs